rllib/evaluation/tests/test_rollout_worker.py

```diff
@@ -197,34 +197,6 @@ def test_no_step_on_init(self):
                 lambda: config.build(),
             )
 
-    def test_query_evaluators(self):
```

Comment:
Why was this test removed?

Reply:
it was using foreach_worker_with_index() API that nobody else seems to care about.
but it's ok, I brought it back. looks like a useful API to have just in case.